src/app/core/components/header/header-nav/header-nav.component.ts

```diff
@@ -8,13 +9,17 @@ import { Component, OnInit } from '@angular/core';
 export class HeaderNavComponent implements OnInit {
   isSearchOpen = false;
   showMenu: Boolean = false;
+  count = [];
```

Comment:
Maybe it would be better for `count` param to be a number, not an array (because name `count` suggests it's a number, otherwise you can name it differently)